Makefile

```diff
@@ -17,18 +17,11 @@ export PACKAGE_NAME=$(PACKAGE_TITLE)-$(VERSION)
 # Defaults that are likely to vary by platform. These are cleanly separated so
 # it should be easy to maintain altered values on platform-specific branches
 # when the values aren't overridden by the script invoking the Makefile
-DEFAULT_APPLY_PATCHES=yes
 DEFAULT_DESTDIR=$(CURDIR)/BUILD/
 DEFAULT_PREFIX=$(PACKAGE_NAME)
 DEFAULT_SYSCONFDIR=PREFIX/etc/$(PACKAGE_TITLE)
 DEFAULT_SKIP_TESTS=no
-
-
-# Whether we should apply patches. This only makes sense for alternate packaging
-# systems that know how to apply patches themselves, e.g. Debian.
-ifndef APPLY_PATCHES
-APPLY_PATCHES=$(DEFAULT_APPLY_PATCHES)
-endif
+DEFAULT_NEW_BUILD=no
```

Comment:
DEFAULT_NEW_BUILD - its not clear what is NEW_BUILD and when this needs to be set.

Could you add a comment as to why this is being set?
When set to yes, what should be the behavior?

Looks like when set to yes, it downloads from nexus.
Literal DEFAULT_NEW_BUILD doesn't make it obvious, may be you can use a better name here.
How about DEFAULT_PULL_ARTIFACTS ?

Reply:
I don't remember why these ended up written this way (probably because way back when I made a bad choice) but we don't need to separately define the `DEFAULT_XYZ` variable and then use it to override the default, we can just inline these into the line that sets the `XYZ` variable.

I agree on naming -- could be clearer. `PULL_ARTIFACTS` or something around "prebuilt" maybe.